### ISSUE TITLE
[TTY]: VAMC - Locations should have TTY: 711 links for deaf/hard-of-hearing users

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23578]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
As a user who might want to make a Text Telephone (TTY) call, I would like to have that link available on the VAMC Location pages. If I don't have TTY set up on my smart phone, I would have to exit the VAMC location page to call 7-1-1.

## Details
I've been working with Meli on the VA Mobile team to standardize TTY language and affordances. While doing some research into another application, I took a look at our VAMC pages. There's an opportunity to improve our views by adding TTY links for deaf/hard-of-hearing users. Screenshots attached below.

## Acceptance Criteria

- [ ] TTY links are added to `/locations` page
- [ ] TTY links are added to location detail views
- [ ] Links are ideally built using our [Telephone component](https://design.va.gov/storybook/?path=/docs/components-telephone--default)
- [ ] TTY text the correct content pattern. **TTY:** is text only, and **711** is linked text. The Telephone component is encouraged because it adds a correct `aria-label` automatically.

## Environment
* https://staging.va.gov/pittsburgh-health-care/locations/
* https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/

## Proposed Solution (if known)

```javascript
<div>
  // Previous content
  TTY: <Telephone contact="711" />
</div>
```

## WCAG or Vendor Guidance (optional)
* [Making actions keyboard accessible by using keyboard event handlers](https://www.w3.org/WAI/GL/wiki/Making_actions_keyboard_accessible_by_using_keyboard_event_handlers_with_WAI-ARIA_controls)

## Screenshots or Trace Logs
![Screen Shot 2021-04-21 at 11 50 28 AM](https://user-images.githubusercontent.com/934879/115592395-c80dac00-a298-11eb-8a45-fb22cc0f1f33.png)

---
![Screen Shot 2021-04-21 at 11 50 38 AM](https://user-images.githubusercontent.com/934879/115592407-ce038d00-a298-11eb-8115-5c7c9a19bb1e.png)

**Comment by joshkimux** [comment]
@laflannery for awareness

**Comment by laflannery** [comment]
I also noticed that the existing phone numbers are not using the `<va-telephone>` component. Can we include that in this issue or should I make a separate task to update those?

**Comment by Agile6MSkinner** [comment]
@thejordanwood Do we have designs for this?
